src/main/java/seedu/contax/model/ImportCsv.java

```diff
@@ -0,0 +1,89 @@
+package seedu.contax.model;
+
+import static seedu.contax.commons.util.AppUtil.checkArgument;
+import static seedu.contax.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.io.File;
+
+public class ImportCsv {
```

Comment:
I suggest changing the name of this class, it does not look like a noun. Use the IntelliJ refactor function.

```suggestion
public class IndexedCsvFile {
```

Reply:
thanks i also agree lol will change!